app/src/features/Batches/BatchList.tsx

```diff
@@ -19,68 +19,75 @@ import BatchCancel from "./BatchCancel";
 import BatchErrors from "./BatchErrors";
 
 const useStyles = makeStyles((theme) => ({
+  accordions: {
+    margin: theme.spacing(0, 5, 8, 5),
+  },
+  accordion: {
+    borderRadius: theme.shape.borderRadius,
+    marginBottom: theme.spacing(0.5),
+    "&.Mui-expanded": {
+      margin: 0,
+      marginBottom: theme.spacing(0.5),
+    },
+    "&:before": {
+      display: "none",
+    },
+  },
+  accordionSummary: {
+    borderBottom: "none",
+    transition: "none",
+    "&.Mui-expanded": {
+      minHeight: 56,
+      borderBottom: `1px solid ${theme.palette.background.default}`,
+    },
+  },
   accordionSummaryContent: {
-    width: "100%",
     display: "flex",
     justifyContent: "space-between",
     alignItems: "center",
     padding: theme.spacing(1, 0),
-  },
-  accordionSummary: {
-    margin: 0,
+    flexWrap: "wrap",
     minHeight: 56,
-    "& > .MuiAccordionSummary-content": {
-      minHeight: 56,
-      margin: 0,
-      transition: "none",
-      "&.Mui-expanded": {
-        minHeight: 56,
-      },
-    },
-    "&.MuiAccordionSummary-root": {
-      borderBottom: `1px solid ${theme.palette.background.default}`,
+    margin: 0,
+    transition: "none",
+    "&.Mui-expanded": {
```

Comment:
I feel like this whole "&.Mui-expanded" is not useful anymore as you're setting `minHeight` & `margin` with the same values right before

Reply:
When the accordion is expanded, it's erasing the props you have when you set the "not" expanded